Majasdarbs/05-majasdarbs/if-else/switch.js

```diff
@@ -0,0 +1,22 @@
+const getDay = (darbaDiena) => {
+switch (darbaDiena) {
```

Comment:
you can use prettier to format all files. Will have spaces that will make code more readable